iwyu_include_picker.cc

```diff
@@ -1304,14 +1305,16 @@ vector<string> IncludePicker::GetCandidateHeadersForFilepathIncludedFrom(
     const string& included_filepath, const string& including_filepath) const {
   vector<string> retval;
   const string quoted_includer = ConvertToQuotedInclude(including_filepath);
```

Comment:
> I would have used its own path with the reasoning that this results in the
> filename only unless there is an -I that matches. 

This seems like a really good reason to use the file's own path. I didn't pick up on that last time I asked why it was there. But it introduces sufficient implicit coupling to `ConvertToQuotedInclude` that it needs a comment to explain *why* it's necessary to pass that file path.